packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/test.sh

```diff
@@ -3,14 +3,14 @@
 . ./upgrade-test-scripts/env_setup.sh
 
 # provision pool has right balance 
-test_val $(agd query bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | first | .amount ') "19000000"
+test_val $(agd query bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | first | .amount ') "18750000"
 
 test_wallet_state "$USER1ADDR" upgraded "user1 wallet is upgraded"
 test_wallet_state "$GOV1ADDR" revived "gov1 wallet is revived"
 test_wallet_state "$GOV2ADDR" revived "gov2 wallet is revived"
 test_wallet_state "$GOV3ADDR" revived "gov3 wallet is revived"
 
-test_val $(agd q vstorage children published.vaultFactory.managers.manager0.vaults -o json | jq -r '.children | length') 2 "we only have two vaults"
+test_val $(agd q vstorage children published.vaultFactory.managers.manager0.vaults -o json | jq -r '.children | length') 3 "we only have three vaults"
```

Comment:
"only" suggests a limit.
```suggestion
test_val $(agd q vstorage children published.vaultFactory.managers.manager0.vaults -o json | jq -r '.children | length') 3 "we have three vaults"
```